src/kaleidoscope/driver/i2cmaster.cpp

```diff
@@ -12,7 +12,7 @@
 #include <inttypes.h>
 #include <compat/twi.h>
 
-#include "kaleidoscope/hardware/ez/ErgoDox/i2cmaster.h"
+#include "kaleidoscope/driver/i2cmaster.h"
```

Comment:
We have our own i2c master code (starting in KeyboardioScanner).
I'd prefer that we not lift the EZ i2c master to be a global driver without some really good reason.

Reply:
The main reason for having EZ's i2c is that I didn't have the time to port it to the one in `KeyboardioScanner`. I'll move it back to `hardware/ez/ErgoDox`, and will eventually port it. You're right, we shouldn't send the message that this is some kind of global i2c driver - it isn't.